modyn/supervisor/internal/pipeline_executor/pipeline_executor.py

```diff
@@ -203,6 +208,10 @@ def run(self) -> None:
             self._serve_online_data(self.state, self.logs)
 
         self._done(self.state, self.logs)
+
+        self._post_pipeline_evaluation_checkpoint(self.state, self.logs)
```

Comment:
I don't understand yet what exactly you plan to checkpoint here.

Reply:
The information about the different triggers/trainings/models. E.g. what the first and last timestamp of a training request was, to what model_id it lead etc.
In the next PR that should become much clearer.
The goal behind this is to allow to restart the pipeline at this point. This will be useful when we adjust the evaluation logic and cannot guarantee that it won't fail. With this we don't need to run training again